deps/bugsnag-plugin-android-unreal/src/main/java/com/bugsnag/android/unreal/UnrealPlugin.java

```diff
@@ -130,6 +118,10 @@ static void notify(String name, String message, StackTraceElement[] stacktrace,
     if (client == null || name == null) {
       return;
     }
+    if (discardClasses != null && discardClasses.contains(name)) {
```

Comment:
Is this due to missing behavior in the bugsnag-android implementation ...?

Reply:
oh right - forgot about that! Thanks.

Reply:
So did I in my first attempt 🙃